### PR TITLE
Add Liquor Mart (Manitoba)

### PR DESCRIPTION
**Pull request by andrewpmk**
Add Liquor Mart (liquor stores in Manitoba, Canada) https://www.liquormarts.ca/

### CODE REVIEW

File: data/brands/shop/alcohol.json [hunk 1]
```diff
@@ -450,6 +450,25 @@
...
+      "matchNames": [
+        "manitoba liquor & lotteries corporation",
+        "manitoba liquor control commission",
+        "mlcc",
+        "mllc"
+      ],
+      "tags": {
+        "brand": "Liquor Mart",
+        "brand:wikidata": "Q124030913",
+        "name": "Liquor Mart",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/alcohol.json:468]:
```suggestion
        "name": "Liquor Mart",
        "operator": "Manitoba Liquor & Lotteries Corporation",
        "operator:short": "MBLL",
        "operator:wikidata": "Q29467564",
```

File: data/brands/shop/alcohol.json [hunk 2]
```diff
@@ -450,6 +450,25 @@
         "shop": "alcohol"
       }
     },
+    {
+      "displayName": "Liquor Mart",
```
Comment (AntiCompositeNumber) [inline data/brands/shop/alcohol.json:454]: Would "Manitoba Liquor Mart" be more appropriate?

Comment (andrewpmk) [reply data/brands/shop/alcohol.json:454]: Signage just says "Liquor Mart" as does website https://www.liquormarts.ca/

Example: https://www.openstreetmap.org/node/4927513512 Look at the Bing street view imagery. Sign is just Liquor Mart. Mapper mapped this as "Carman Liquor Mart" though.

Comment (AntiCompositeNumber) [reply data/brands/shop/alcohol.json:454]: The website does use Manitoba Liquor Mart in some places, but I'll agree that Liquor Mart is used primarily in branding. I'll merge as-is.

**Review by AntiCompositeNumber — APPROVED** [review]

**Pull request merged**